src/components/deliverables/Deliverables.vue

```diff
@@ -112,7 +113,8 @@
                       v-on:click="rowClicked(index)" 
                       v-bind:class="{ highlight: rowSelected(item.primaryFileId) }"
                       >
-                   <td>{{item.itemName}}</td>
+                    <td>{{item.collection}}</td>
```

Comment:
Let's call the field item.collectionName, to be accurate (the label being "Collection" is fine).